functional_tests/scylla_operator/conftest.py

```diff
@@ -113,6 +113,8 @@ def fixture_db_cluster(tester: ScyllaOperatorFunctionalClusterTester):
         original_scylla_config_map = scylla_config_map
     original_scylla_cluster_spec = tester.db_cluster.get_scylla_cluster_plain_value('/spec')
 
+    if dataset_size := tester.db_cluster.params.get("k8s_scylla_operator_functional_test_dataset_size"):
```

Comment:
you are calling `dataset_size`, but it's actually a name (or a key in a dict). 

dataset_name, probably would be better.

Reply:
Actually it's size: small, medium, big

Reply:
see my below comment, about giving the names as they are in `load_inventory`

Reply:
changed